styx-cli/pom.xml

```diff
@@ -69,6 +69,11 @@
       <artifactId>jackson-annotations</artifactId>
       <version>${jackson.version}</version>
     </dependency>
+      <dependency>
+          <groupId>com.spotify</groupId>
+          <artifactId>styx-client</artifactId>
+          <version>1.0.6-SNAPSHOT</version>
```

Comment:
yeah, the versions for some of the dependencies, like argparse4j and jackson are already defined in the parent pom under deps management, so no need to specify them here.